### DISCUSSION TITLE
Change the header {name}

### DISCUSSION THREAD

**Discussion by rizqijune** [heysagnik/Linkees#38] [Q&A] [answered]
Hi, can anyone show me how to change the name in the header? I cant find any json, or yaml to change it. Already checked it on the header, index as well. But still no idea.

What method is used in this package?
I'm a noob

**Comment by rizqijune** [comment, marked as answer]
Nvm, I build this wrongly. Thats why you dont copy & paste the code and just blindly build it. Thank you for this wonderfull package!

**Reply by heysagnik** [reply]
So is the issue fixed?

**Comment by Corentin5** [comment]
Hello, I would like to know how can I change {name} in Header using the node_module ?

**Reply by heysagnik** [reply]
@taksuparth please guide him

**Reply by taksuparth** [reply]
@Corentin5 which version of Linkees are you using? We have added support for the following in our version 1.0.2 

Please refer to the following example: https://codesandbox.io/s/linkees-gji6dl
